.github/workflows/build.yaml

```diff
@@ -1,19 +1,26 @@
-name: Build tokens
+name: Build Tokens
 
 on:
-  workflow_dispatch:
-  workflow_call:
```

Comment:
We might still want the ability to test the build manually on specific branches

Reply:
Reverted. Thanks!